buildrunner/steprunner/__init__.py

```diff
@@ -82,6 +82,8 @@ def __init__(
         self.container_labels = container_labels
         # network name is used to identify the network that the build step is running in
         self.network_name = f"{build_runner.build_id}-{step_name}"
+        if build_runner.run_config_file:
+            self.network_name += f"-{os.path.basename(os.path.dirname(os.path.abspath(build_runner.run_config_file)))}"
```

Comment:
Why not generate a random ID here? What if they are in subdirectories that match the name?

Reply:
that sounds better...